src/renderer/lib/startDolphin.ts

```diff
@@ -0,0 +1,72 @@
+import { DolphinType } from "./directories";
+import { remote } from "electron";
+import { randomBytes } from "crypto";
+import { assertDolphinInstallation, openDolphin } from "./downloadDolphin";
+import { assertPlayKey } from "./playkey";
+import * as fs from "fs-extra";
+
+import path from "path";
+
+export interface ReplayComm {
```

Comment:
Can we rename this to not be short-hand? Something like `ReplayCommunication` or something? I feel like exported interfaces and function names shouldn't be shortened.

Reply:
gotcha